rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/IndexJavaSourceFilesRuleProvider.java

```diff
@@ -76,7 +80,7 @@ private IndexJavaFileIterationOperator()
         @Override
         public void perform(GraphRewrite event, EvaluationContext context, JavaSourceFileModel payload)
         {
-            WindupRoasterWildcardImportResolver.setGraphContext(event.getGraphContext());
+            WindupWildcardImportResolver.setGraphContext(event.getGraphContext());
```

Comment:
Ok, this is fine for now but I'd like to find a cleaner solution later. This isn't a big enough deal to hold up the PR.